cmake/scripts/rustc_wrapper.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+REQUIRED_ENVS=("MESATEE_PROJECT_ROOT" "MESATEE_BUILD_ROOT")
+for var in "${REQUIRED_ENVS[@]}"; do
+    [ -z "${!var}" ] && echo "Please set ${var}" && exit -1
+done
+
+exec rustc "$@" --remap-path-prefix=${HOME}/.cargo=/cargo_home --remap-path-prefix=${MESATEE_PROJECT_ROOT}=/mesatee_src --remap-path-prefix=${MESATEE_BUILD_ROOT}=/mesatee_build
```

Comment:
Can you add a comment before this line saying that why do we need to replace `debug-prefix-map`.

Reply:
done